lookout/style/typos/corrector_manager.py

```diff
@@ -25,7 +25,8 @@ def get(self, source: str) -> TyposCorrector:
         """
         obj = self.correctors.get(source)
         if obj is None:
-            obj = TyposCorrector(processes_number=1).load(source)
+            obj = TyposCorrector().load(source)
+            obj.threads_number = 1
```

Comment:
```suggestion
            obj.processes_number = 1
```
This parameter was changed in the merged commits.

Reply:
@EgorBu please merge this suggestion